### PR TITLE
Translate Accessibility

### PR DESCRIPTION
**Pull request by ekafyi**
Hi, here’s my PR. Some notes:
* I used a flexible translation approach (eg. altering sentence structure to one that sounds more natural in Indonesian) without changing the meaning.
* My references are, in order of priority: this repo’s “Glosarium”, other accepted PRs, KBBI, large/established sources (eg. Google & Firefox’s Indonesian webites), search results.

I listed specific recurrent words & phrases and my translation strategies [in this gist](https://gist.github.com/ekaonthenet/26c85a8c2335aa672a9db97ca7446d10); might be useful for you to review and advise accordingly.

Thanks!

### CODE REVIEW

File: index.mdx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
```
Comment (gedeagas) [inline index.mdx]: Hi @ekaonthenet, thank you for your contribution. 

May I ask what is this file about ?

And also please refrain from committing yarn lock file to the repo. Thank You.

Comment (ekaonthenet) [reply index.mdx]: My bad, that file should not be here, wrong drag n drop.
Removing it and `yarn.lock` now.

**Comment by grikomsn** [comment]
Since the `yarn.lock` and other unrelated files are commited long time ago, I recommend submitting another pull request or force-pushing the branch.

**Comment by ekafyi** [comment]
Hi @resir014 @grikomsn,
Shall I add `yarn.lock` back to this branch regardless or is https://github.com/reactjs/id.reactjs.org/pull/89 sufficient?

**Comment by grikomsn** [comment]
IMO, in order to resolve the conflicts, initiate a soft git reset to commit df09630a7c77dba8a1126784f9e7347c04d13df0, then re-commit the "Accessibility" translation progress like what I did on #89.

Or you can improve the translation on #89 (which I have given @ekafyi and @resir014 permission to commit) and close this PR.

**Comment by ekafyi** [comment]
> IMO, in order to resolve the conflicts, initiate a soft git reset to commit [df09630](https://github.com/reactjs/id.reactjs.org/commit/df09630a7c77dba8a1126784f9e7347c04d13df0), then re-commit the "Accessibility" translation progress like what I did on #89.

Hi @grikomsn, I've done this solution (git reset then recommit). Please kindly check.

**Pull request merged**